src/service/extension-location.js

```diff
@@ -21,14 +21,11 @@ import {getMode} from '../mode';
  * Calculate the base url for any scripts.
  * @param {!Location} location The window's location
  * @param {boolean=} isLocalDev
- * @param {boolean=} isTest
  * @return {string}
  */
-function calculateScriptBaseUrl(location, isLocalDev, isTest) {
+function calculateScriptBaseUrl(location, isLocalDev) {
   if (isLocalDev) {
-    if (isTest || isMax(location) || isMin(location)) {
-      return `${location.protocol}//${location.host}/dist`;
-    }
+    return `${location.protocol}//${location.host}/dist`;
```

Comment:
I really dislike `isLocalDev` being passed here. At the minimum, it kills the DCE. I suppose we can live with it, at least for this PR. But if it's easy to remove, let's do.